backend/models.py

```diff
@@ -0,0 +1,66 @@
+
+# Create your models here.
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.contrib.postgres.fields import ArrayField
+
+import uuid
+
+
+class User(AbstractUser):
+    githubUrl = models.URLField(max_length=400)
+    # Using: username, password, first_name, last_name, email inherited from Abstractuser
+
+
+class Post(models.Model):
+    postId = models.UUIDField(
+        primary_key=True, default=uuid.uuid4, editable=False, unique=True)
+    author = models.ForeignKey(User, on_delete=models.CASCADE)
+    title = models.CharField(max_length=50)
+    timestamp = models.DateTimeField(auto_now_add=True)
+    content = models.TextField()
+    # default posts are public
+    isPublic = models.BooleanField(default=True)
+
+
+class PostAccess(models.Model):
+    accessChoices = [
+        ("PUBLIC", 'Public'),
+        ("PRIVATE", 'Private'),
+        ("FRIENDS", 'Friends'),
+        ("FOF", 'FoF'),
+    ]
+    # post = models.ForeignKey(Post, on_delete=models.CASCADE) # remove
+    visibility = models.CharField(max_length=10, choices=accessChoices, default="PUBLIC")
+    # contains users that are able to access the post
+    visible_to = ArrayField(models.CharField(max_length=500), blank=True, default=list)
```

Comment:
What if the user has a lot of friends. Won't the max_length here cause an issue with that?